2214/2/battler.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'bad_words_counter'
+require_relative 'words_in_round_counter'
+require_relative 'battle'
+
+class Battler
+  BATTLES_FOLDER = 'Battles'.freeze
+  attr_reader :name, :battles
+  def initialize(name, battles)
+    @name = name
+    @battles = battles
+  end
+
+  def number_of_battles
+    @battles.size
+  end
+
+  def number_of_bad_words
+    BadWordsCounter.new(@battles).count
```

Comment:
What if we remove `BadWordsCounter` and just add a `Battle#bad_words_count` method? ;-)